alphabase/io/tempmmap.py

```diff
@@ -90,6 +92,41 @@ def array(shape: tuple, dtype: np.dtype) -> np.ndarray:
             offset=offset
         ).reshape(shape)
 
+def map_array(hdf_file: str) -> np.ndarray:
```

Comment:
map_array looks like a dict/map array, which is ambiguious, maybe mmap_array is better.

Reply:
Sure sounds look a good idea.